src/language-js/print/assignment.js

```diff
@@ -43,6 +43,26 @@ function printAssignment(
     ? print(rightPropertyName, { assignmentLayout: layout })
     : "";
 
+  return printAssignmentWithLayout(
+    path,
+    options,
+    print,
+    leftDoc,
+    operator,
+    rightDoc,
+    layout,
+  );
+}
+
+function printAssignmentWithLayout(
```

Comment:
A `printAssignmentWithLayout` function that allows you to pass rightDoc and layout directly.The logic is the same as the original `printAssignment`.